src/AWSSDK.MobileAnalytics.MobileAnalyticsManager.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>net45;net35</TargetFrameworks>
     <DefineConstants>$(DefineConstants);BCL</DefineConstants>
-    <Version>3.5.0.0</Version>
+    <Version>3.6.0.0</Version>
```

Comment:
Unsure if we only want to increment by 1, or keep matching SDK with 3.7

Reply:
This is fine